Makefile

```diff
@@ -80,14 +81,18 @@ test-courier:
 	echo "bundle dir: $(BUNDLE_DIR_VERSION)"
 	export BUNDLE_DIR_VERSION; docker build -f build/Dockerfile.test-courier -t test-courier . --build-arg dir=$(BUNDLE_DIR_VERSION)
 
-container-push: container-push-operator container-push-registry
+container-push: container-push-operator container-push-registry container-push-must-gather
 
 container-push-operator:
 	docker push $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG)
 
 container-push-registry:
 	docker push $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG)
 
+container-push-must-gather:
+	echo "env: IMAGE_TAG="$(IMAGE_TAG)" IMAGE_REGISTRY="$(IMAGE_REGISTRY)" MUST_GATHER_IMAGE="$(MUST_GATHER_IMAGE)" OPERATOR_IMAGE="$(OPERATOR_IMAGE)" REGISTRY_IMAGE="$(REGISTRY_IMAGE)
```

Comment:
I guess this was for debugging? Can be removed then.

Reply:
removed.